infra-templates/etcd-operator/0/README.md

```diff
@@ -0,0 +1,16 @@
+etcd-operator
```

Comment:
Lower administrative overhead since everything is in the same repo. I've done this sort of thing with other projects such as the OpenShift integration prototype. Upstream puts the k8s manifests in an [example dir](https://github.com/coreos/etcd-operator/tree/master/example) - it's similar to that.